Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -5210,6 +5210,7 @@ void CClientPed::Respawn(CVector* pvecPosition, bool bRestoreState, bool bCamera
             float         fTargetRotation = m_pPlayerPed->GetTargetRotation();
             unsigned char ucInterior = GetInterior();
             unsigned char ucCameraInterior = static_cast<unsigned char>(g_pGame->GetWorld()->GetCurrentArea());
+            std::string   sCameraGoogleEffect = CLuaCameraDefs::GetCameraGoggleEffect();
```

Comment:
This is bad. CLuaCameraDefs(as well as other CLua*Defs classes) is only responsible for the interaction with Lua. It must not be available from other classes.

Reply:
so where I move it to class CCameraSA?

Reply:
Maybe something like this?
```
bool bOldNightVision = g_pMultiplayer->IsNightVisionEnabled();
bool bOldThermalVision = g_pMultiplayer->IsThermalVisionEnabled();

...

g_pMultiplayer->SetNightVisionEnabled(bOldNightVision, true)
g_pMultiplayer->SetThermalVisionEnabled(bOldThermalVision, true)
```